beacon-chain/blockchain/process_block_test.go

```diff
@@ -41,103 +41,6 @@ import (
 	logTest "github.com/sirupsen/logrus/hooks/test"
 )
 
-func TestStore_OnBlock(t *testing.T) {
```

Comment:
This test does no longer test `onBlock` but rather the previous helpers that are tested elsewhere